apps/ecommerce/frontend/src/components/Field/Wrappers/MultipleResources.tsx

```diff
@@ -0,0 +1,58 @@
+import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+import { FieldAppSDK } from '@contentful/app-sdk';
+import { ResourceField } from 'components/Field/ResourceField';
+import mockValue from 'helpers/mockValue';
+import ResourceFieldProvider from 'providers/ResourceFieldProvider';
+import type { ExternalResourceLink } from 'types';
+
+const MultipleResources = () => {
+  const sdk = useSDK<FieldAppSDK>();
+
+  const handleAddContent = () => {
+    // TODO: Update this function to add a new resource(s) to the list
+    const newValue = mockValue(sdk);
+    const resourceArray = sdk.field.getValue();
+
+    if (resourceArray) {
+      sdk.field.setValue([...resourceArray, newValue]);
+    } else {
+      sdk.field.setValue([newValue]);
+    }
+  };
+
+  const handleRemove = (index: number) => {
+    const resourceArray = sdk.field.getValue();
+
+    const newValue = resourceArray.filter((obj: ExternalResourceLink, i: number) => i !== index);
+    sdk.field.setValue(newValue);
```

Comment:
Slight inconsistency here with the single value. For a single, the remove makes this value undefined. But here, on the last item in the array, it'll just be an empty array.

Do we care about making this distinction?